Makefile

```diff
@@ -11,10 +11,10 @@ C_SRCS = stub_lseek.c stub_write.c stub_close.c stub_open.c
 OBJS = $(ASM_SRCS:%.s=%.o) $(C_SRCS:%.c=%.o)
 
 %.o: %.s
-	as6502 --core=6502 --list-file=$(@:%.o=obj/%.lst) -o obj/$@ $<
+	as6502 --core=45gs02 --list-file=$(@:%.o=obj/%.lst) -o obj/$@ $<
```

Comment:
I do not want to change this to 45gs02 as it would prevent all other platforms from working. This project is for all Commodore style platforms and 6502 is supported by all.

When the library is built for the Calypsi product, this `Makefile` is not used. Instead the internal build tool uses a directory scan to search for source files and then apply variant specific options such as `--target=45gs02`. Thus, when built for a specific target to be included in the prebuilt board support the appropriate options are used.

The `Makefile` is mainly for those that wants to build their own variant. Feel free to use 45gs02 for your variant build, but it should not be part of the PR.

If the Makefile is to support 45gs02, it would need different build variants in the Makefile.

Reply:
Oops, I meant to revert this one. Took care of that.